From 6f5e40032c16172e5797151e7a7e2275e5eb0738 Mon Sep 17 00:00:00 2001
From: Johannes Schindelin <johannes.schindelin@gmx.de>
Date: Fri, 2 Jun 2017 17:54:57 +0200
Subject: [PATCH 1/8] RunTest: skip a couple more tests on Windows

Some of these require a larger stack (which we cannot force on Windows),
some are locale dependent (when the locale mechanism is completely
different from what PCRE2's tests expect).

Signed-off-by: Johannes Schindelin <johannes.schindelin@gmx.de>
---
 RunTest | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/RunTest b/RunTest
index ae32f5d..23985ea 100755
--- a/RunTest
+++ b/RunTest
@@ -441,8 +441,8 @@ if [ $do0  = no -a $do1  = no -a $do2  = no -a $do3  = no -a \
    ]; then
   do0=yes
   do1=yes
-  do2=yes
-  do3=yes
+  do2=no
+  do3=no
   do4=yes
   do5=yes
   do6=yes
@@ -454,14 +454,14 @@ if [ $do0  = no -a $do1  = no -a $do2  = no -a $do3  = no -a \
   do12=yes
   do13=yes
   do14=yes
-  do15=yes
+  do15=no
   do16=yes
   do17=yes
   do18=yes
   do19=yes
   do20=yes
   do21=yes
-  do22=yes
+  do22=no
   do23=yes
   do24=yes
   do25=yes
-- 
2.39.0.rc0.windows.1

